crypto/eots/error.go

```diff
@@ -4,17 +4,17 @@ import (
 	ecdsa_schnorr "github.com/decred/dcrd/dcrec/secp256k1/v4/schnorr"
 )
 
-// ErrorKind identifies a kind of error.  It has full support for errors.Is
+// KindError identifies a kind of error.  It has full support for errors.Is
```

Comment:
```suggestion
// KindOfError identifies a kind of error. It has full support for errors.Is
```
Curious why linter is not happy with `ErrorKind`?

Reply:
apparently convention for error types is to end name with suffix `Error` (underlying `ecdsa_schnorr.ErrorKind` does not follow this convention)